dspace-server-webapp/src/test/java/org/dspace/app/rest/EPersonRestRepositoryIT.java

```diff
@@ -110,6 +135,34 @@ public void createTest() throws Exception {
         // TODO cleanup the context!!!
     }
 
+    @Test
+    public void createAnonAccessDeniedTest() throws Exception {
+        context.turnOffAuthorisationSystem();
```

Comment:
This *must* be followed by a `context.restoreAuthSystemState();` in this same method after you are done creating content

Reply:
This has been included now